designer/client/conditions-edit.js

```diff
@@ -30,44 +31,49 @@ class ConditionsEdit extends React.Component {
       <div className='govuk-body'>
         {!condition ? (
```

Comment:
I would use `(condition &&)` and `(!condition &&)`. ternaries are hard to read when they're multiline

Reply:
Done